src/Equations/acoustic/Model/Datastructures.h

```diff
@@ -0,0 +1,107 @@
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @section DESCRIPTION
+ **/
+
+#ifndef MODEL_ACOUSTIC_DATASTRUCTURES_H_
+#define MODEL_ACOUSTIC_DATASTRUCTURES_H_
+
+#include "Model/CommonDatastructures.h"
+#include "generated_code/init.h"
+#include "generated_code/kernel.h"
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <string>
+
+namespace seissol::model {
+struct AcousticMaterial : Material {
+  static constexpr std::size_t NumQuantities = 4;
+  static constexpr std::size_t NumberPerMechanism = 0;
+  static constexpr std::size_t Mechanisms = 0;
+  static constexpr MaterialType Type = MaterialType::Acoustic;
+  static constexpr LocalSolver Solver = LocalSolver::CauchyKovalevski;
+  static inline const std::string Text = "acoustic";
+  static inline const std::array<std::string, NumQuantities> Quantities = {
+      "p", "v1", "v2", "v3"};
+
+  double lambda;
+
+  double getLambdaBar() const override { return lambda; }
+
+  double getMuBar() const override { return 0.0; }
+
+  AcousticMaterial() = default;
+  AcousticMaterial(const double* materialValues, int numMaterialValues) {
+    assert(numMaterialValues == 2);
+
+    this->rho = materialValues[0];
+    this->lambda = materialValues[1];
+  }
+
+  ~AcousticMaterial() override = default;
+
+  void getFullStiffnessTensor(std::array<double, 81>& fullTensor) const override {
+
+    auto stiffnessTensorView =
+        seissol_general::init::stiffnessTensor::view::create(fullTensor.data());
+    stiffnessTensorView.setZero();
+    stiffnessTensorView(0, 0, 0, 0) = lambda;
+    stiffnessTensorView(0, 0, 1, 1) = lambda;
+    stiffnessTensorView(0, 0, 2, 2) = lambda;
+    stiffnessTensorView(1, 1, 0, 0) = lambda;
+    stiffnessTensorView(1, 1, 1, 1) = lambda;
+    stiffnessTensorView(1, 1, 2, 2) = lambda;
+    stiffnessTensorView(2, 2, 0, 0) = lambda;
+    stiffnessTensorView(2, 2, 1, 1) = lambda;
+    stiffnessTensorView(2, 2, 2, 2) = lambda;
+  }
```

Comment:
Please write in comments the stiffness Tensor you used. If it is still in line with the stress-velocity formulation used for elastic materials or if you changed it accordingly for the pressure-velocity formulations(the key difference would be a change of sign in some places). We have to make sure it is consistent and easy to understand for your successor.